cms/envs/aws.py

```diff
@@ -159,7 +159,7 @@
 ]
 
 LOG_DIR = ENV_TOKENS['LOG_DIR']
-DATA_DIR = ENV_TOKENS.get('DATA_DIR', DATA_DIR)
+DATA_DIR = path(ENV_TOKENS.get('DATA_DIR', DATA_DIR))
```

Comment:
I don't think the other `envs` that doesn't reuse this have the same problem, right?